### DISCUSSION TITLE
Gcode file subfolder search

### DISCUSSION THREAD

**Discussion by owliwar** [mainsail-crew/mainsail#1811] [Help]
I cant seem to find a way to search for a file in subfolders? i have to manually open each folder and perform a search and its very difficult with multiple folders and subfolders. I'm I missing something? I could'nt find anywhere on advanced search or something like that 

Thanks

**Comment by meteyou** [comment]
right now, we use the Vuetify data-table search function. With this function its only possible to search in the current directory, because only these objects are available for this search.
